server/routes/termdb.boxplot.ts

```diff
@@ -163,17 +163,25 @@ function setDescrStats(boxplot: BoxPlotData, sortedValues: number[]) {
 	const sd = Math.sqrt(s / (sortedValues.length - 1))
 	const squareDiffs = sortedValues.map(x => (x - mean) ** 2).reduce((a, b) => a + b, 0)
 	const variance = squareDiffs / (sortedValues.length - 1)
+
+	/** boxplot_getvalue won't return these values for small # of samples
+	 * Calculate value subset for tooltips. */
```

Comment:
Instead of recalculating below by copying code from `boxplot_getvalue()`, it's less redundant and less error-prone to support an optional second argument in `boxplot_getvalue()`, that could override the minimum 5 value requirement. However, I'm not sure if it's still valid to render a boxplot in our case  with less than 5 values, maybe @xzhou82 could verify. Also, calculating quartiles with less than 5 values seems to not be related to [the linked issue](https://github.com/stjude/proteinpaint/issues/2403), where there are 800+ values, so this extra code seems optional to me since the fix is really within the LegendRender.ts updates?
```js
export function boxplot_getvalue(lst, minNumValues=5) {
	/* ascending order
    each element: {value}
    */
	const l = lst.length
	if (l < minNumValues) {
		// less than minNumValues/items, won't make boxplot
		return { out: lst }
	}
```

Reply:
i agree here it shouldn't re-calculate when there're less than 5 data points --  just do not show these stats on the client is okay

Reply:
This is fixed.